src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -207,5 +208,137 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 F.CloseMethod(F.Block(F.Return(F.Field(F.This(), _property.BackingField))));
             }
         }
+
+        private sealed class InitAccessorSymbol : SynthesizedInstanceMethodSymbol
```

Comment:
`SythesizedInitAccessorSymbol`?

Reply:
since this is a private class I prefer the short name, and it's unlikely to get confused with a different `init` accessor :)